openwpm/storage/cloud_storage/s3_storage.py

```diff
@@ -1,5 +1,103 @@
+import logging
+from typing import Any, Set
+
+import pyarrow.parquet as pq
+from pyarrow.lib import Table
+from s3fs import S3FileSystem
+
 from ..arrow_storage import ArrowProvider
+from ..storage_providers import TableName, UnstructuredStorageProvider
+
+
+class S3StructuredProvider(ArrowProvider):
+    """This class allows you to upload Parquet files to S3.
+
+    Inspired by the old S3Aggregator structure the S3StructuredProvider
+    will by default store into
+    base_path/visits/table_name in the given bucket.
```

Comment:
```suggestion
    S3StructuredProvider will by default store into
    base_path/visits/table_name in the given bucket.
```

I don't think it's necessary to reference old code here.